autofaiss/external/distributed.py

```diff
@@ -0,0 +1,367 @@
+"""
```

Comment:
let's put this file in indices folder since it's not exposed to the user

Reply:
Agree

Reply:
Yeah, I hesitated where should I put it the best, now I got the answer.